### PR TITLE
Accsec 26769

### PR DESCRIPTION
**Pull request by elkinnarvaez**
Hi there! This PR mostly includes all the code refactoring I did because the code wasn't very easily readable at first. It also includes some betterments pointed out from the testing team. The related Jira tickets of those betterments are:

- https://issues.corp.twilio.com/browse/ACCSEC-26886
- https://issues.corp.twilio.com/browse/ACCSEC-26885
- https://issues.corp.twilio.com/browse/ACCSEC-26878

As Jose is in PTO, I am just assigning you as a reviewer. Let me know what you think, and thanks in advance.

### CODE REVIEW

File: templates.json [hunk 1]
```diff
@@ -312,8 +312,8 @@
     },
     {
       "id": "verify-sna",
-      "name": "verify-sna",
-      "description": "Sample Backend Silent Network Auth"
+      "name": "Verify Silent Network Auth Sample Backend",
+      "description": "Starts and checks SNA verifications and shows their status in the dashboard as they come in"
```
Comment (mejiagarcia) [inline templates.json]: what about:

`
Backend to communicate with Verify API. This backend starts and validates login request against the Verify API by using the SNA channel functionality. The results will be presented on the UI dashboard

`

Just a suggestion, maybe the current description is too technical for someone that doesn't have the context.

File: verify-sna/functions/verify-start.js [hunk 2]
```diff
@@ -26,46 +26,47 @@
...
+const { createVerification } = require(assets['/services/verifications.js']
+  .path);
+const { detectMissingParams } = require(assets['/services/helpers.js'].path);
 
 // eslint-disable-next-line consistent-return
 exports.handler = async function (context, event, callback) {
   const response = new Twilio.Response();
   response.appendHeader('Content-Type', 'application/json');
 
+  const missingParams = detectMissingParams(
```
Comment (mejiagarcia) [inline verify-sna/functions/verify-start.js:43]: I noticed that these parameters `countryCode` and `phoneNumber` are being used both on `verify-start.js` and `verify-check.js`. Any chance to make them constants?

File: templates.json (hunk 1 above)
Comment (elkinnarvaez) [reply templates.json]: Yes sounds a lot better, thanks

File: verify-sna/functions/verify-start.js [hunk 3]
```diff
@@ -26,46 +26,47 @@
...
+    ['countryCode', 'phoneNumber'],
+    event
+  );
+  if (missingParams.length > 0) {
```
Comment (mejiagarcia) [inline verify-sna/functions/verify-start.js:47]: This logic is also repeated in the `verify-check.js` file, any chance to move it to a helper method and reuse it?

**Review by chriss-m240 — APPROVED** [review]

**Pull request merged**